.circleci/config.yml

```diff
@@ -38,7 +38,7 @@ jobs:
             # Copy the repository to this volume
             docker cp /root/girder_worker configs:/cfg
             # Run the integration tests after installing requirements
-            docker run --volumes-from configs --network container:gw_integration_test_girder python:2.7.13 /bin/bash -c "cd /cfg/girder_worker/tests/integration; pip install -r requirements.txt; pytest -v -n 4"
+            docker run --volumes-from configs --network container:gw_integration_test_girder python:2.7.13 /bin/bash -c "cd /cfg/girder_worker/tests/integration; pip install -r requirements.txt; pytest -v -n 4 -m 'not docker'"
```

Comment:
Could you explain this change? Are we no longer running docker integration tests?

Reply:
So there are currently two sets of docker tests. The "heirloom" ones that run on Travis, these are still being run and the new ones that test the docker_run task. The problem is the new test infrastructure runs on Circle 2.0, so we need to figure our if its possible to run docker in docker etc. The is the added complication of using named pipe, with Circle 2.0, there isn't really a host file system on which to create these pipes. So for now I disabled these tests, I know this is not ideal and its definitely a goal to get these run some how, it might be a reason to move back to VM based containers, in fact I just looked at there website and they advertise this as a way of  "Avoid the “Docker in Docker” difficulties ...".

Reply:
Ok, sounds good. Can we just add a `# TODO ` line above this that says "re-enable docker tests in CI" or something along those lines?